File: source/serial_adapter/source/serial_adapter.py
# pip3 install websocket-client
from websocket import create_connection

def onMessage(ws, message):
    print(message)

ws = create_connection("ws://192.168.0.8:9091/", on_message=onMessage)
ws.send("positionUpdate(&**&)2 2")
ws.close()

File: source/serial_adapter/source/serial_adapter.js
var activityManagerSocket = new WebSocket("ws://192.168.0.8:9091");

activityManagerSocket.onmessage = function (event) {
    console.log(event.data);
  }

activityManagerSocket.send("positionUpdate(&**&)2 0");
await sleep(2000);
activityManagerSocket.send("positionUpdate(&**&)2 2");
await sleep(2000);
activityManagerSocket.send("positionUpdate(&**&)2 0");